security/security_tools.md

```diff
@@ -0,0 +1,179 @@
+Report on Dynamic and Static Analysis Security Tools
+====================================================
+
```

Comment:
Currently it's confusing to me that report seems to provide recommendations, and then there are a lot of tools described that aren't introduced in the "recommendations" section. Maybe adding an introduction stating that this report describes a number of tools that were investigated, including some that aren't included in the "recommendations" section, would be helpful. 

It might be helpful for each tool that's not part of the recommendations section, to add a bit about why it's not recommended. But I wouldn't block on that.

Reply:
Okay I think it would be helpful to list the different categories here and summarize what each category is, including whether it is automated (and can be put in CI) or manual. There are summaries below each category, and they can be moved up here to give an initial overview of what will be encountered in this document. I still feel unsure whether some of these tools just can't be integrated into CI at all, so being very clear about that would be helpful.